### ISSUE TITLE
Graphical MCQ - Checkboxes are not visible when multiple answers selectable

### ISSUE THREAD

**Issue by Tamayuzu** [LIT-EIA/adapt_authoring#32]
When the selection of multiple answers is enabled, the checkboxes that replace the radio buttons are white (same as the background color) so they are not visible to users.

.gmcq-component .gmcq-item label .gmcq-answer-icon.checkbox
.gmcq-component .gmcq-item label.selected .gmcq-answer-icon.checkbox
